catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -209,15 +218,16 @@ public void setAttributeOverrides(List<String> attributeOverrides) {
     private void attemptAddRoutes() {
         LOGGER.debug("Attempting to add routes for content directory monitor watching {}",
                 monitoredDirectory);
-        RouteBuilder routeBuilder = createRouteBuilder();
         try {
+            RouteBuilder routeBuilder = createRouteBuilder();
             verifyContentCamelComponentIsAvailable();
             camelContext.addRoutes(routeBuilder);
             setRouteCollection(routeBuilder);
         } catch (Exception e) {
             LOGGER.warn("{} {}",
                     "Unable to configure Camel route.",
-                    "This content directory monitor will be unusable.");
+                    "This content directory monitor will be unusable.",
```

Comment:
Camel's API doesn't give us a lot to work with there. `Throws: Exception - if the routes could not be created for whatever reason`. Hopefully the actual exception would have something useful.